src/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -275,17 +271,22 @@ export const OuiBreadcrumbs: FunctionComponent<OuiBreadcrumbsProps> = ({
           )}
         </OuiInnerText>
       );
-    }
 
-    let wrapper = <div className={breadcrumbWrapperClasses}>{link}</div>;
-
-    if (isFirstBreadcrumb) {
-      const breadcrumbWallClasses = classNames('ouiBreadcrumbWall', {
-        'ouiBreadcrumbWall--single': isLastBreadcrumb,
-      });
-
-      wrapper = <div className={breadcrumbWallClasses}>{wrapper}</div>;
-    }
+    const breadcrumbWallClasses = isFirstBreadcrumb
+      ? classNames('ouiBreadcrumbWall', {
+          'ouiBreadcrumbWall--single': isLastBreadcrumb,
+        })
+      : breadcrumbWrapperClasses;
+
+    const wrapper = (
+      <div className={breadcrumbWallClasses}>
+        {isFirstBreadcrumb ? (
+          <div className={breadcrumbWrapperClasses}>{link}</div>
+        ) : (
+          link
+        )}
+      </div>
+    );
```

Comment:
I feel like this is difficult to fully understand. I feel like something like this would make more sense

```suggestion
    const breadcrumbWallClasses = classNames('ouiBreadcrumbWall', {
      'ouiBreadcrumbWall--single': isFirstBreadcrumb && isLastBreadcrumb,
    });
    
    const wrapper = <div className={breadcrumbWrapperClasses}>{link}</div>;
    const wall = isFirstBreadcrumb ? <div className={breadcrumbWallClasses}>{wrapper}</div> : wrapper;
```

And then line 291 would use `wall` instead of `wrapper`

Reply:
Hi @BSFishy,

Makes totally sense what you say. Thanks for the suggestion. Just update the PR with the required improvements.